base/io.jl

```diff
@@ -525,15 +525,7 @@ function read(s::IO, nb::Integer = typemax(Int))
     return resize!(b, nr)
 end
 
-"""
-    readstring(stream::IO)
-    readstring(filename::AbstractString)
-
-Read the entire contents of an I/O stream or a file as a string.
-The text is assumed to be encoded in UTF-8.
-"""
-readstring(s::IO) = String(read(s))
-readstring(filename::AbstractString) = open(readstring, filename)
```

Comment:
This method is still relevant, isn't it? I didn't see it redefined elsewhere in the diff but maybe I missed it. In this case it'd just be
```julia
read(filename::AbstractString, ::Type{String}) = open(io -> read(io, String), filename)
```